From: Dmitry Shachnev <mitya57@debian.org>
Date: Sat, 16 Jul 2022 10:42:43 +0800
Subject: disable dav1d support on mipsel/mips64el

dav1d does not support MIPS, and there is no config directory for it.
Forwarded: no
Last-Update: 2021-02-14
---
 src/3rdparty/chromium/media/media_options.gni | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/3rdparty/chromium/media/media_options.gni b/src/3rdparty/chromium/media/media_options.gni
index 32c021e..acacfcd 100644
--- a/src/3rdparty/chromium/media/media_options.gni
+++ b/src/3rdparty/chromium/media/media_options.gni
@@ -100,7 +100,7 @@ declare_args() {
   # are combined and we could override more logging than expected.
   enable_logging_override = !use_jumbo_build && is_chromecast
 
-  enable_dav1d_decoder = !is_android && !is_ios
+  enable_dav1d_decoder = !is_android && !is_ios && target_cpu != "mipsel" && target_cpu != "mips64el"
 
   # Enable browser managed persistent metadata storage for EME persistent
   # session and persistent usage record session.
